### DISCUSSION TITLE
🚀 Feature Request: Multiple post widgets - [ED-12189]

### DISCUSSION THREAD

**Discussion by Dreamland-MR** [elementor/elementor#21718] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

As discussed with Rica from the Tier 2 Technical Support Specialist team, it would be nice, to be able to have multiple post widgets with separate paginations ("Numbers" or "Prev+Next"), so if you change the set of one post widget, it won't change the setting of the other widgets.
 

### Describe the solution you'd like

I would like to have separate pagination on multiple post widgets

**Comment by nicholaszein** [comment]
Hello @Dreamland-MR!

Thank you for submitting your request, and apologies for the delay, we are currently organizing all feature requests in one place, and we have a backlog to handle! 🙏 

I'm happy to announce this feature will soon be possible with the new Loop Grid widget. Keep an eye on our [Roadmap](https://elementor.com/roadmap/)!

Kind regards
